third-party/java/intellij/get_jars.sh

```diff
@@ -59,43 +114,15 @@ if [[ -z "$SRC_DIR" ]]; then
   fi
 fi
 
-
 # ====================================================================
 #  Copy and/or stub jars
 # ====================================================================
 printf "Copying IntelliJ jars from %q\n" "$SRC_DIR"
+copy_or_stub_jars "$SRC_DIR" ${#INTELLIJ_LIBS_TO_COPY[@]} ${INTELLIJ_LIBS_TO_COPY[@]} ${#INTELLIJ_LIBS_TO_STUB[@]} ${INTELLIJ_LIBS_TO_STUB[@]}
 
-SHOULD_FAIL=false
-for lib in "${ALL_LIBS[@]}"; do
-  if ! [[ -e "$SRC_DIR/$lib" ]]; then
-    >&2 echo "ERROR:  Can't find required library $lib in $SRC_DIR"
-    SHOULD_FAIL=true
-  fi
-done
-if $SHOULD_FAIL ; then
-  echo "Aborting."
-  exit 1
-fi
-
-for lib in "${LIBS_TO_COPY[@]}"; do
-  echo "Copying $lib..."
-  cp "$SRC_DIR/$lib" "$DEST_DIR/$lib"
-done
-for lib in "${LIBS_TO_STUB[@]}"; do
-  if [[ -e "$DEST_DIR/$lib" ]]; then
-    printf "Removing previous $lib...  "
-    rm "$DEST_DIR/$lib"
-  fi
-  echo "Stubbing $lib..."
-  if ! (cd "$DEST_DIR" && 2>/dev/null buck run //src/com/facebook/buck/jvm/java/abi:api-stubber "$SRC_DIR/$lib" "$DEST_DIR/$lib") ; then
-    echo "ERROR:  Failed to make stubbed copy of $lib"
-    SHOULD_FAIL=true
-  fi
-done
-if $SHOULD_FAIL ; then
-  echo "Aborting."
-  exit 1
-fi
+ANDROID_SRC_DIR="$SRC_DIR/../plugins/android/lib"
+printf "Copying IntelliJ Android jars from %q\n" "$ANDROID_SRC_DIR"
+copy_or_stub_jars "$ANDROID_SRC_DIR" ${#ANDROID_LIBS_TO_COPY[@]} ${#ANDROID_LIBS_TO_STUB[@]} ${ANDROID_LIBS_TO_COPY[@]} ${ANDROID_LIBS_TO_STUB[@]}
```

Comment:
It looks like the arguments are wrong:

> copy_or_stub_jars "$ANDROID_SRC_DIR" ${#ANDROID_LIBS_TO_COPY[@]} ${#ANDROID_LIBS_TO_STUB[@]} ${ANDROID_LIBS_TO_COPY[@]} ${ANDROID_LIBS_TO_STUB[@]}

should be

> copy_or_stub_jars "$ANDROID_SRC_DIR" ${#ANDROID_LIBS_TO_COPY[@]} ${ANDROID_LIBS_TO_COPY[@]} ${#ANDROID_LIBS_TO_STUB[@]} ${ANDROID_LIBS_TO_STUB[@]}

The format is: number of libs, libs, number of stubs, stubs. But in your case it's: number of libs, number of stubs, libs, stubs.

Ideally, to catch these kinds of errors you would need to have some kind of separators in the list of arguments, but it's up to you to use that.